test/browser.replay.recorder.test.js

```diff
@@ -148,8 +148,8 @@ describe('Recorder', function () {
 
       // Event count includes the custom end event
       expect(mockSpan.addEvent.calledThrice).to.be.true;
-      expect(mockSpan.setAttribute.calledOnce).to.be.true;
-      expect(mockSpan.setAttribute.calledWith('rollbar.replay.id', testReplayId)).to.be.true;
+      expect(mockSpan.setAttributes.calledOnce).to.be.true;
+      expect(mockSpan.setAttributes.calledWith({'rollbar.replay.id': testReplayId})).to.be.true;
```

Comment:
Oh, but how do you test it if you use the `set` keyword instead of making a `setAttributes` function?